banzai/images.py

```diff
@@ -40,13 +40,11 @@ def __init__(self, pipeline_context, filename=None, data=None, header=None,
 
         self.request_number = header.get('REQNUM')
 
-        self.site = header.get('SITEID')
-        self.instrument = header.get('INSTRUME')
-        if self.site is not None and self.instrument is not None:
-            self.telescope_id = dbs.get_telescope_id(self.site, self.instrument,
-                                                     db_address=pipeline_context.db_address)
-        else:
-            self.telescope_id = None
+        self.telescope = dbs.get_telescope(self.header)
+        if self.telescope is not None:
+            self.site = self.telescope.site
+            self.instrument = self.telescope.instrument
+
```

Comment:
Would it make sense to fall back to header values for `site` and `instrument` if a telescope isn't found?

Reply:
I agree, good idea. how about something like
self.site = header.get('SITEID')
self.instrument = header.get('instrument')
prior to the if self.telescope is not None call